Makefile

```diff
@@ -44,9 +46,9 @@ ci.lint-ruby: ## Run Rubocop with results formatted for CI
 
 .PHONY: ci.cucumber
 ci.cucumber: ## Run the Cucumber specs
-	docker-compose run web /bin/sh -c 'mkdir $(RESULTS_PATH) && \
-		bundle exec cucumber --format junit --out $(RESULTS_PATH)'
-	$(call copy_test_results)
+	docker-compose run web /bin/sh -c 'mkdir $(CUCUMBER_RESULTS_PATH) && \
+		bundle exec cucumber --format junit --out $(CUCUMBER_RESULTS_PATH)'
+	$(call copy_to_host,$(CUCUMBER_RESULTS_PATH))
```

Comment:
TIL you could do this 🎉

Reply:
but you'll have to be careful with the spaces.
`$(call copy_to_host, $(CUCUMBER_RESULTS_PATH))` would add a space before the parameter when invoked 😬